webpack/scenes/ContentViews/Details/Filters/Rules/ContainerTag/AddEditContainerTagRuleModal.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Modal, ModalVariant, Form, FormGroup, TextInput, ActionGroup, Button } from '@patternfly/react-core';
+import { addCVFilterRule, editCVFilterRule, getCVFilterRules } from '../../../ContentViewDetailActions';
+
+const AddEditContainerTagRuleModal = ({
+  onClose, filterId, selectedFilterRuleData = { name: '', id: undefined },
+}) => {
+  const { name, id } = selectedFilterRuleData;
+  const dispatch = useDispatch();
+  const [tagName, setTagName] = useState(name);
+  const [saving, setSaving] = useState(false);
+  const isEditing = name && id;
+
+  const onSave = () => {
+    setSaving(true);
+    if (isEditing) {
+      dispatch(editCVFilterRule(
+        filterId,
+        {
+          id,
+          name: tagName,
+          editMode: true,
+          working: false,
+          previous: {
+            id,
+            name,
+            editMode: true,
+            working: false,
+          },
+        },
+        () => dispatch(dispatch(getCVFilterRules(filterId))),
```

Comment:
Is the double dispatch wrapping necessary?

Reply:
oh! that is not intentional!